docs/dev/api.rst

```diff
@@ -61,6 +61,14 @@ Core API Endpoints
   :statuscode 400: invalid JSON sent, see error message for details.
   :statuscode 401: invalid authorization. See error message for details.
 
+Statistics API Endpoints
+^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Let's add a small paragraph here about what stats are, how often they're updated and how they are in beta right now.